apps/prairielearn/src/ee/lib/lti13.ts

```diff
@@ -229,13 +229,30 @@ export class Lti13Claim {
       'http://purl.imsglobal.org/vocab/lis/v2/membership#Instructor',
       'http://purl.imsglobal.org/vocab/lis/v2/membership/Instructor#TeachingAssistant',
       'http://purl.imsglobal.org/vocab/lis/v2/system/person#User'
+     ]
+    Student roles
+    [
+      'http://purl.imsglobal.org/vocab/lis/v2/institution/person#Instructor',
+      'http://purl.imsglobal.org/vocab/lis/v2/institution/person#Student',
+      'http://purl.imsglobal.org/vocab/lis/v2/membership#Learner',
+      'http://purl.imsglobal.org/vocab/lis/v2/system/person#User'
+    ]
+    Designer roles
+    [
+      'http://purl.imsglobal.org/vocab/lis/v2/institution/person#Instructor',
+      'http://purl.imsglobal.org/vocab/lis/v2/membership#ContentDeveloper',
+      'http://purl.imsglobal.org/vocab/lis/v2/system/person#User'
     ]
     */
 
     let role_instructor = this.roles.some((val: string) =>
-      ['Instructor', 'http://purl.imsglobal.org/vocab/lis/v2/membership#Instructor'].includes(val),
+      [
```

Comment:
I removed the `Instructor` Context short-form and added the long-form ContentDeveloper.

The short-form use is deprecated based on https://www.imsglobal.org/spec/lti/v1p3#lis-vocabulary-for-context-roles